app/models/merchant_employee.rb

```diff
@@ -0,0 +1,4 @@
+class MerchantEmployee < ApplicationRecord
+  belongs_to :merchant
+  belongs_to :user
+end
```

Comment:
The merchant model here is distinct from the user model? looks like there's a many to many relationship here